apps/ensindexer/test/ids.test.ts

```diff
@@ -66,6 +94,18 @@ describe("ids", () => {
       });
     });
 
+    describe("parseResolverId", () => {
+      it("should parse chain-scoped format", () => {
+        const node = namehash("vitalik.eth");
+        const resolverId = makeResolverId(CHAIN_ID, zeroAddress, node);
+        expect(parseResolverId(resolverId)).toEqual({
+          chainId: CHAIN_ID,
+          address: zeroAddress,
+          node,
+        });
+      });
```

Comment:
I can see multiple `describe("parseResolverId", () => {` code blocks including single test cases. What do you think about merging all test cases for `parseResolverId` into a single code block?